Themis/Views/Assessment/CorrectionSidebar/ProblemStatementCellView.swift

```diff
@@ -11,7 +11,7 @@ import MarkdownUI
 
 struct ProblemStatementCellView: View {
     @Environment(\.colorScheme) var colorScheme
-    var problemStatement: String?
+    @Binding var submission: SubmissionForAssessment?
```

Comment:
The problemstatement has to be a binding otherwise it will not be updated when a new submission is loaded. However it is not possible to have conditional chaining with bindings therefore making it impossible to only pass the submission.

Reply:
It is important to note that I'm talking about the explicit behaviour that the problemstatement is reloaded when another submission is loaded and not onAppear.

Reply:
This works:
```
problemStatement: Binding(
                    get: { submission?.problemStatement },
                    set: { submission?.problemStatement = $0 }
                )
```

Reply:
Using the custom binding works, although its really stupid you'd have to use something like this...